src/pyhf/infer/calculators.py

```diff
@@ -303,6 +303,20 @@ def _false_case():
             )
         return teststat
 
+    def pvalues(self, teststat, sig_plus_bkg_distribution, b_only_distribution):
+        '''calculate pvalues.'''
```

Comment:
I'll review this thoroughly tomorrow morning, but just to hit on the docs related things, as these methods are in the public API we should have full docstrings with examples for them (for both `pvalues` and `expected_pvalues` both here and in the `EmpiricalDistribution` ).

Reply:
I should add that this PR is super nice to have! :)